@here/harp-mapview/lib/MapView.ts

```diff
@@ -2200,6 +2261,23 @@ export class MapView extends THREE.EventDispatcher {
         this.drawFrame();
     }
 
+    /**
+     * Need to be called if the [[MapView.camera]] position, rotation, quaternion properties are
+     * changed manually.
+     */
+    updateCamera(lookAtDistance?: number) {
+        if (lookAtDistance === undefined) {
+            this.m_lookAtDistance = this.getTarget()
```

Comment:
Hmm, this is a bit fishy. getTarget will return a vector with length m_lookAtDistance so m_lookAtDistance will effectively never change if the user is moving the camera and no target point is hit.
I think you need to compute the lookAtDistance from the distance to ground (like we do in the zoomlevel getter)